### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by FredrikAugust**
Updated design, and added a landing page.

The theme for the design can easily be modified by editing the file `styling/theme.ts`.

### CODE REVIEW

File: src/components/LandingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+interface ILandingPage {
+  className?: string;
+}
+
+const BrandTitle = styled.h1`
+  font-family: "Montserrat", sans-serif;
+  text-transform: uppercase;
+  font-size: 4em;
+  font-weight: 300;
+  color: ${theme.contrast};
```
Comment (nicolajn) [inline src/components/LandingPage.tsx]: Would be nice to have properties like font-familiy, size etc. in the theme as well for reusability and consistent design throughout the site.

Comment (FredrikAugust) [reply src/components/LandingPage.tsx]: Font-family is defined in the global styling file, here I'm just overriding as I want the brand to pop a bit more. As for the font-size, I believe it is so individual for each component that it's no use in putting it in a global file.

File: src/components/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+  }
+
+  p {
+    color: ${theme.contrast};
+    font-size: 1.2em;
+  }
+
+  /* Media queries */
+  @media screen and (max-width: 600px) {
```
Comment (nicolajn) [inline src/components/LandingPage.tsx]: We should discuss which breakpoint we should use on the entire app and put it in the theme, maybe(?)

Comment (FredrikAugust) [reply src/components/LandingPage.tsx]: Yep, just used 600 here for testing how it responded with hot-reloading. Spoiler alert: it worked well.

**Pull request merged**